lisa/sut_orchestrator/azure/common.py

```diff
@@ -1148,7 +1150,7 @@ def save_console_log(
             )
         screenshot_raw_name.unlink()
 
-    log_response = requests.get(diagnostic_data.serial_console_log_blob_uri)
+    log_response = requests.get(diagnostic_data.serial_console_log_blob_uri, timeout=60)
```

Comment:
Is it the default value from the requests package?

Reply:
The default is it will not timeout and hang until something else stops it. I picked 60 as a very conservative value.
https://requests.readthedocs.io/en/latest/user/advanced/#timeouts